src/l10n/locales.ts

```diff
@@ -9,7 +9,7 @@ type MarketName =
   | 'Danmark'
   | 'Denmark'
 type LanguageLabel = 'Sv' | 'En' | 'No' | 'Da'
-type HtmlLang = 'sv' | 'en' | 'no' | 'da'
+type HtmlLang = 'sv-se' | 'en-se' | 'no-no' | 'en-no' | 'da-dk' | 'en-dk'
```

Comment:
I'm not sure if it makes a difference but shouldn't country part be upper case?

https://www.techonthenet.com/js/language_tags.php

Reply:
Not sure it matters either but let's play it safe :D